Sources/Testing/Support/FileHandle.swift

```diff
@@ -397,7 +399,7 @@ extension FileHandle {
 
   /// Is this file handle a pipe or FIFO?
   var isPipe: Bool {
-#if SWT_TARGET_OS_APPLE || os(Linux)
+#if SWT_TARGET_OS_APPLE || os(Linux) || os(WASI)
```

Comment:
As with TTYs, is it actually possible for a file to be a pipe?

Reply:
WASI Preview 1 doesn't have fifo nor pipe but Preview 2 does! https://github.com/WebAssembly/wasi-filesystem/blob/fe0ebbcb8bab1ee1f860812c25a6cc83b725e47b/wit/types.wit#L52

Reply:
Okay, works for me!